raml-parser-2/pom.xml

```diff
@@ -70,6 +70,11 @@
             <artifactId>xmlschema-core</artifactId>
             <version>2.2.1</version>
         </dependency>
+	<dependency>
+            <groupId>org.opengis.cite.xerces</groupId>
+            <artifactId>xercesImpl-xsd11</artifactId>
```

Comment:
This is the Xerces implementation for XSD 1.1 support which relaxes the Unique Particle Attribution when there's an `any` element and a more specific rule for `additionalProperties:`.